chall03/vde-dios.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import sys
+
+
+def snail(arg):
+    top = len(arg[1])
+    ini = 1
+    while (ini <= top):
+        i = ini
+        while (i <= top):
+            print(arg[ini][i - 1] if (ini is 1 and i is ini) else ", %s" % arg[ini][i - 1], end = '')
+            i += 1
+        i = ini + 1
+        while (i < top):
+            print(", %s" % arg[i][top - 1], end = '')
+            i += 1
+        if (ini < top):
+            i = top
+            while ( i >= ini):
+                print(", %s" % arg[top][i - 1], end = '')
+                i -= 1
+            i = top - 1
+            while ( i > ini):
+                print(", %s" % arg[i][ini - 1], end = '')
+                i -= 1
+        ini += 1
+        top -= 1
+
+
+def check_args():
+    for c in range(len(sys.argv) - 1):
+        if len(sys.argv) - 1 != len(sys.argv[c + 1]) or len(sys.argv) > 21:
+            return None
+        for c in sys.argv[c + 1]:
+            if not c.isnumeric():
+                return None
+    return sys.argv
+
+
+def main():
+    arg = check_args()
+    if arg is None:
+        return print("usage: %s <login> <a-A-Z string>" % sys.argv[0])
```

Comment:
I think `sys.exit("usage: %s <login> <a-A-Z string>" % sys.argv[0])` would have been better.

You are using usage message from the last challenge.